index.html

```diff
@@ -4669,8 +4669,8 @@ <h2 id="AudioBufferSourceNode">
           This interface represents an audio source from an in-memory audio
           asset in an <code>AudioBuffer</code>. It is useful for playing audio
           assets which require a high degree of scheduling flexibility, for
-          instance, playing back in rhythmically-perfect ways. If
-          sample-accurate playback of network- or disk-backed assets is
+          instance, playing back musical events that are rhythmically perfect.
+          If sample-accurate playback of network- or disk-backed assets is
```

Comment:
My thought was since the spec already used the word "rhythmically" then we must already be talking about music, and the phrase seemed to be an example rather than spec language.

I can remove this language to talk about accurate timing instead.